test/t8_test_cmesh_copy.c

```diff
@@ -23,9 +23,27 @@
 #include <t8_cmesh.h>
 #include "t8_cmesh/t8_cmesh_trees.h"
 #include "t8_cmesh/t8_cmesh_partition.h"
+#include <t8_eclass.h>
+
+#define T8_CMESH_TEST_NUM_COMMS 2
+#define T8_CMESH_TEST_DIMS 3
+#define T8_CMESH_MIN_DIM 1
```

Comment:
You use this Macro only in one function. I think, this should be a local variable.